README.md

```diff
@@ -41,13 +41,65 @@ $runner->addExtension(new \mageekguy\atoum\ruler\extension($script));
 
 ## Use it
 
-Examples :
+The extension adds a `--filter` option to atoum. This line now appears on the atoum help :
+
+```
+--filter: Filters tests to execute. For example 'not("featureA" in tags) and namespace = "foo\bar"'
+```
+
+You can now filter your tests using any [\Hoa\Ruler filter](https://github.com/hoaproject/Ruler).
+
+Those variables are available in the filter :
+* method
+* class
+* namespace
+* testedclass
+* testedclassnamespace
+* tags (as an array)
+
+## Examples
+
+Run all tests except those who have the "needsDatabase" tag :
+```
+./vendor/bin/atoum -d tests --filter 'not ("needsDatabase" in tags)'
+```
+
+Run all tests with a method named "testMethod1" :
 ```
-./vendor/bin/atoum -d tests --filter 'not ("unSuperTagAuNiveauDeLaMethode1" in tags)'
 ./vendor/bin/atoum -d tests --filter 'method = "testMethod1'
+```
+
+
+Run all tests with a method named "testMethod1" (using an an array to contain the list of filtered methods) :
+```
 ./vendor/bin/atoum -d tests --filter 'method in ["testMethod1"]'
+```
+
+
+You can also defined more complex filters like this : run all tests tagged tagged "needsDatabase" and with method "testClass1" or with a method "testClass" :
+```
+./vendor/bin/atoum --filter '("needsDatabase" in tags and method = "testClass1") or (method = "testClass")'
+```
+
+
+Runs tthe test with the "mageekguy\atoum\ruler\tests\units\testClass1" classname :
+```
 ./vendor/bin/atoum -d tests --filter 'class = "mageekguy\atoum\ruler\tests\units\testClass1'
+```
+
+
+Run all tests in the "mageekguy\atoum\ruler\tests\units" namespace :
+```
 ./vendor/bin/atoum -d tests --filter 'namespace = "mageekguy\atoum\ruler\tests\units'
+```
+
+
+Run the tests that tests the "mageekguy\atoum\ruler\testClass1" class :
```

Comment:
“Run the tests that test the”, it's plural, so “that tests” is wrong, no need for a “s”.